recipes/gdal/post_3.5.0/conanfile.py

```diff
@@ -0,0 +1,759 @@
+            self.requires("freexl/1.0.6")
+
+        if self.options.with_geos:
+            self.requires("geos/3.11.0")
+
+        if self.options.with_gif:
+            self.requires("giflib/5.2.1")
+
+        if self.options.with_gta:
+            self.requires("libgta/1.2.1")
+
+        if self.options.with_hdf4:
+            self.requires("hdf4/4.2.15")
+
+        if self.options.with_hdf5:
+            self.requires("hdf5/1.13.1")
+
+        if self.options.with_heif:
+            self.requires("libheif/1.12.0")
+
+        if self.options.with_kea:
+            self.requires("kealib/1.4.14")
+
+        if self.options.with_libdeflate:
+            self.requires("libdeflate/1.12")
+
+        if self.options.with_libiconv:
+            self.requires("libiconv/1.17")
+
+        if self.options.with_libjpeg:
+            self.requires("libjpeg/9d")
+
+        if self.options.with_libkml:
+            self.requires("libkml/1.3.0")
+
+        if self.options.with_libtiff:
+            self.requires("libtiff/4.3.0")
+
+        if self.options.with_lz4:
+            self.requires("lz4/1.9.3")
+
+        if self.options.with_mongocxx:
+            self.requires("mongo-cxx-driver/3.6.6")
+
+        if self.options.with_netcdf:
+            self.requires("netcdf/4.8.1")
+
+        if self.options.with_odbc:
+            self.requires("odbc/2.3.9")
+
+        if self.options.with_openjpeg:
+            self.requires("openjpeg/2.5.0")
+
+        if self.options.with_openssl:
+            self.requires("openssl/1.1.1o")
+
+        if self.options.with_pcre:
+            self.requires("pcre/8.45")
+
+        if self.options.with_pcre2:
+            self.requires("pcre2/10.40")
+
+        if self.options.with_pg:
+            self.requires("libpq/14.2")
+
+        if self.options.with_png:
+            self.requires("libpng/1.6.37")
+
+        if self.options.with_podofo:
+            self.requires("podofo/0.9.7")
+
+        if self.options.with_poppler:
+            self.requires("poppler/21.07.0")
+
+        if self.options.with_proj:
+            self.requires("proj/9.0.0")
+
+        if self.options.with_qhull:
+            self.requires("qhull/8.0.1")
+
+        if self.options.with_sqlite3:
+            self.requires("sqlite3/3.38.5")
+
+        if self.options.with_webp:
+            self.requires("libwebp/1.2.2")
+
+        if self.options.with_xerces:
+            self.requires("xerces-c/3.2.3")
+
+        if self.options.with_xml2:
+            self.requires("libxml2/2.9.14")
+
+        if self.options.with_zlib:
+            self.requires("zlib/1.2.12")
+
+        if self.options.with_zstd:
+            self.requires("zstd/1.5.2")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version],
+                destination=self._source_subfolder, strip_root=True)
+
+    @functools.lru_cache(1)
+    def _configure_cmake(self):
+        cmake = CMake(self)
+
+        if "fPIC" in self.options and self.options.fPIC:
```

Comment:
AFAIK, we delete it as shared because it must be true to work 🤔 havent touched this in a while

```suggestion
        if self.options.get_safe("fPIC", True):
```